merlin/models/tf/blocks/dlrm.py

```diff
@@ -108,9 +108,12 @@ def DLRMBlock(
                 "The bottom_block is required by DLRM when "
                 "continuous features are available in the schema"
             )
-        con = ContinuousFeatures.from_schema(con_schema)
-        bottom_block = con.connect(bottom_block)  # type: ignore
-        interaction_inputs = ParallelBlock({"embeddings": embeddings, "bottom_block": bottom_block})
+        interaction_inputs = InputBlockV2(
```

Comment:
It may be the case (demonstrated here) that use of the `InputBlockV2` doesn't make the code any clearer or simpler. I would argue that in this case it would be clearer to keep the construction of the ParallelBlock as it exists already.

Reply:
I agree @oliverholworthy. The code without `InputBlockV2` is indeed clearer. 
I think the main benefit objective here is to use the new `EmbeddingTable`, so maybe we should replace the `EmbeddingFeatures.from_schema()` by `Embeddings(schema)`.
What do you think?

Reply:
Just like you do [here](https://github.com/NVIDIA-Merlin/models/pull/619) :)